README.md

```diff
@@ -1,2 +1,34 @@
-# schema
-JSON schemas for Score files
+# JSON schemas for Score files
+
+| version | file |
+| --- | --- |
+| v1-beta1 | score-v1b1.json |
+
+## Embed schemas into project
+
+Add Score schemas into projects with `git subtree add` command:
+
+```
+git subtree add \
+  --prefix schemas \
+  git@github.com:score-spec/schema.git main \
+  --squash
+```
+
+> **Note:** To avoid storing the entire history of the subproject in the main repository make sure to include `--squash` flag.
```

Comment:
```suggestion
> **Note:** To avoid storing the entire history of the sub-project in the main repository, make sure to include `--squash` flag.
```
nitpicking 😬

Reply:
fixed